ru.js

```diff
@@ -0,0 +1,747 @@
+module.exports = `
+menu.btn.ranked Игра на рейтинг
+menu.btn.shop Магазин

```

Comment:
"magazin" is the correct word for shop in russian, but usually more for a physical grocery/supermarket